packages/core/src/snapshot.js

```diff
@@ -292,7 +292,7 @@ export function createSnapshotsQueue(percy) {
     .handle('start', async () => {
       try {
         build = percy.build = {};
-        let { data } = await percy.client.createBuild();
+        let { data } = await percy.client.createBuild({ execType: percy.execType });
```

Comment:
```
percy build:finalize 
```

command uses `createBuild` you'll require to update that as well.

also update tests for this

Reply:
We actually don't need to pass `execType` while calling `createBuild` during `percy build:finalize` As we don't support it in App Percy.
CC @ninadbstack

Reply:
we dont have type check on finalize api irrespective of app/web right ?

Reply:
@ninadbstack Right. We don't have check on finalize API

Reply:
we are passing `execType` in `create_build` call in finalize.js

Reply:
we are not passing projectType in finalize call as we dont use it for app Percy